### ISSUE TITLE
The Reason why Monkey King Aghanims isn't working

### ISSUE THREAD

**Issue by LPSwitch** [ValveSoftware/Dota2-Gameplay#12103]
### Ability name

Wukong's Command

### Description

In regards to this bug, I've noticed that for the Aghanims monkeys to spawn, you need to either:

1.) Have Wukong's Command on cooldown.
2.) Level up Wukong's Command (Works even when WC is off cooldown. Spawns a monkey once, then stops working again). The 4 second spawn interval still applies, so leveling up WC twice in quick succession will not spawn 2 monkeys. You must wait 4 seconds before leveling WC again for another monkey to spawn. This, however, has no practical use whatsoever, so make of it what you will.

Some additional notes:
1.) The Aghs monkeys will stop spawning the moment WC is off cooldown. To enable it again, simply use WC so it goes on cd again.
2.) Death/Respawning has no relation to this bug. Dying and respawning will not fix this issue.
3.) Dropping/Backpacking Aghs then picking it up again will not fix this issue.

TL;DR If you want Aghs monkeys to spawn, just have WC on cd.

This brings a very troublesome issue of being unnecessarily forced to weigh risk/reward on whether or not you want WC to be on cd to maximize farming efficiency but unusable on clashes, or have it off cd and have it readily available for clashes, but drastically reduce farming efficiency.

### Example Match ID (and possibly Timestamp)

7345515580 - 1:59

**Comment by powerofgreed** [comment]
FIX MONKEY KING AGHANIM'S SCEPTER

**Comment by WizeWanter** [comment]
FIX MONKEY KING AGHANIM'S SCEPTER

**Comment by pablopeguero** [comment]
I cant believe it isnt fixed. Is there anywhere else we can direct this problem to? It seems other heros bugs are extremely minor. I cant remeber the last time a hero had his aghs not working.

**Comment by shaileshgarg** [comment]
https://github.com/ValveSoftware/Dota2-Gameplay/issues/12089

**Comment by acke** [comment]
Same issue in game id: 7349110179 - 17:00 and onwards.

**Comment by powerofgreed** [comment]
https://github.com/ValveSoftware/Dota2-Gameplay/issues/12064
